tests/integration/cli/visual_prompting/test_visual_prompting.py

```diff
@@ -8,7 +8,7 @@
 import copy
 import pytest
 
-from otx.cli.registry import Registry
+from src.otx.cli.registry import Registry
```

Comment:
the module name was not changed so you don't need to put 'src.' to the module importing.

Reply:
Fixed.